frontend/src/shared/util/dataset-utils.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { describe, test, expect } from '@jest/globals';
+import { datasetFromS3Uri, datasetComponents } from './dataset-utils';
+import { DatasetType } from '../model';
+
+describe('Test datasetFromS3Uri', () => {
```

Comment:
Should this be in a test directory instead of `src`?

We don't currently have unit-tests. I may have missed a discussion where we decided how we wanted to proceed with unit tests, but if the team hasn't decided collectively on a unit-test standard, I am hesitant to add one-off tests at this point in time, even though adding testing is admirable and should be applauded. Without team buy-in and consistency this adds confusion to the codebase.

Reply:
There are a [few](https://github.com/awslabs/mlspace/blob/main/frontend/src/entities/notebook/notebook.reducer.spec.ts) [tests](https://github.com/awslabs/mlspace/blob/main/frontend/src/entities/model/model.reducer.spec.ts) in the frontend codebase already. It's pretty standard practice to have them in the same directory.